.travis/script.sh

```diff
@@ -7,6 +7,7 @@ set -veuo pipefail
 
 # Lint code.
 flake8 --config flake8.cfg || exit 1
+black --diff . || exit 1
```

Comment:
This should be `black --check --diff . || exit 1` (without `--check` black just outputs a diff without setting an error code when reformatting is required)